cli/ballerina-cli/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -79,6 +71,10 @@ EXAMPLES
         `hello.jar` file in the '<package-root>/target/bin' directory.
           $ bal build
 
+       Build the 'hello' library package with --compile flag. This will generate the
```

Comment:
Better to provide the example `$bal build -c`

Reply:
Fixed in 3f25c0a